Makefile

```diff
@@ -6,6 +6,9 @@ APISERVER_IMG ?= apiserver:latest
 # Docker image name for the mkdocs based local development setup
 IMAGE=onmetal-api/documentation
 
+# kind cluster which should be used
+KIND_CLUSTER ?= kind
```

Comment:
Instead changing the `Makefile` I would suggest to put this into the local setup guide and mention there that the current kind `context` should be set to `kind`.

Reply:
Sure that might be an option. however if someone created a kind cluster with a name different from the defaults it will not work anymore. The consequence is that you need to parse through the entire Makefile and find the places where the kind cluster is referenced. 

If you guys never change the kind content I totally agree that the setup guidelines are enough.

Reply:
The problem is, that we can not - and should not - assume how the users local `kind` setup looks like. That is why we should stick to the default case as much as possible in the beginning.

Reply:
Got it.